pkg/autoscaler/config.go

```diff
@@ -41,6 +43,12 @@ type Config struct {
 	// Target concurrency knobs for different container concurrency configurations.
 	ContainerConcurrencyTargetFraction float64
 	ContainerConcurrencyTargetDefault  float64
+	// TargetUtilization is used for the metrics other than concurrency. This is not
+	// configurable now. Customers can override it by specifying
+	// autoscaling.knative.dev//targetUtilizationPercentage in Revision annotation.
+	TargetUtilization float64
```

Comment:
Do we plan to eventually deprecate `ContainerConcurrencyTargetFraction` in favor of this? Sounds like we should to unify paths as much as possible.

Reply:
I agree. I don't want to expose another flag to confuse customers for next release, which is V1. Added a TODO.